src/Resources/skeleton/controller/ControllerWithTwig.tpl.php

```diff
@@ -13,7 +13,6 @@ class <?= $controller_class_name ?> extends Controller
      */
     public function index()
     {
-        // replace this line with your own code!
-        return $this->render('@Maker/demoPage.html.twig', [ 'path' => str_replace($this->getParameter('kernel.project_dir').'/', '', __FILE__) ]);
+        return $this->render('<?= $twig_file ?>');
```

Comment:
not calling getParameter anymore, this gives back the possibility to extend AbstractController

Reply:
What do you think about passing some variable in as the second arg (to show how that works)?